eclass/tree-sitter-grammar.eclass

```diff
@@ -36,6 +36,47 @@ RESTRICT+=" !test? ( test )"
 # Used to override upstream tag name if tagged differently, e.g. most releases
 # are v${PV} but some are tagged as rust-${PV}.
 
+# @ECLASS_VARIABLE: TS_BINDINGS
+# @PRE_INHERIT
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Array of bindings language to build. Currently only "python" is supported.
+
+_tree_sitter_set_globals() {
+	local binding
+	for binding in "${TS_BINDINGS[@]}"; do
+		case ${binding} in
+			python)
+				DISTUTILS_EXT=1
+				DISTUTILS_OPTIONAL=1
+				DISTUTILS_USE_PEP517=setuptools
+				PYTHON_COMPAT=( python3_{10..12} )
+				inherit distutils-r1
+
+				IUSE+=" python"
+				REQUIRED_USE+=" python? ( ${PYTHON_REQUIRED_USE} )"
+
+				DEPEND+=" python? (
+					${PYTHON_DEPS}
+				)"
+				RDEPEND+=" python? (
+					${PYTHON_DEPS}
+					>=dev-python/tree-sitter-0.21.0[${PYTHON_USEDEP}]
+				)"
+				BDEPEND+=" python? (
+					${DISTUTILS_DEPS}
+					dev-python/wheel[${PYTHON_USEDEP}]
```

Comment:
That should come via `DISTUTILS_DEPS`.

Reply:
Umm, I've checked the eclass once again, and it doesn't add `wheel`, only gpep517 and setuptools.

And before anyone asks, yes, the bindings do need wheel

https://github.com/tree-sitter/tree-sitter-bash/blob/master/setup.py#L6

Reply:
DISTUTILS_USE_PEP517=setuptools

This means that distutils-r1.eclass is responsible to guarantee the build time dependencies for the setuptools "build_meta" backend are satisfied -- this includes "wheel".

In fact it isn't added via DISTUTILS_DEPS, but it is in RDEPEND for setuptools itself. This is technically wrong since a package depending on `import pkg_resources` has to RDEPEND on setuptools but shouldn't be forced to have wheel available too.

Reply:
> Umm, I've checked the eclass once again, and it doesn't add `wheel`, only gpep517 and setuptools. Also verified using `pquery`

Right, because we've moved that dep straight into `dev-python/setuptools`.

> And before anyone asks, yes, the bindings do need wheel
> 
> https://github.com/tree-sitter/tree-sitter-bash/blob/master/setup.py#L6

Hmm, I guess it's fine to depend on it explicitly when it imports stuff from `wheel`.